### PR TITLE
Build-A-DIY-Skycoin-Miner.de.md

### PR DESCRIPTION
**Pull request by kietob**

### CODE REVIEW

File: content/infographics/Build-A-DIY-Skycoin-Miner.de.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
++++
+title = "Bau eines DIY Skyminer [INFOGRAPHIC]"
```
Comment (IamVanCleef) [inline content/infographics/Build-A-DIY-Skycoin-Miner.de.md]: DIY-Skyminers im Deutschen

File: content/infographics/Build-A-DIY-Skycoin-Miner.de.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+categories = [
+    "Infographics",
+]
+[image]
+    url = "img/Build-A-DIY-Skycoin-Miner.jpg"
+    type = "image/jpg"
+    length = 294100
++++
+
+## Um Teil von Skywire zu werden, kannst du deinen eigenen Skyminer bauen. Alles was du brauchst sind einige Fähigkeiten und die folgenden Hardware:
```
Comment (IamVanCleef) [inline content/infographics/Build-A-DIY-Skycoin-Miner.de.md]: Komma nach brauchst

File: content/infographics/Build-A-DIY-Skycoin-Miner.de.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+* x7 Speicher (~ $ 12 pro Stück)
+* x1 Ethernet-LAN-Kabel (~ $ 10)
+* x1 OS (KOSTENLOS)
+* x7 Computermodule (je $ 29)
+* x1 Gehäuse (~ $ 40)
+* x1 Netzteil (~ $ 16)
+
+Gesamtsumme: ~ $ 450
+
+> Lese mehr über die DIY Skyminers hier: https://christianott.co/skycoin_pine64miner_de/
```
Comment (IamVanCleef) [inline content/infographics/Build-A-DIY-Skycoin-Miner.de.md]: Lese mehr über die DIY-Skyminer

File: content/infographics/Build-A-DIY-Skycoin-Miner.de.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+]
+[image]
+    url = "img/Build-A-DIY-Skycoin-Miner.jpg"
+    type = "image/jpg"
+    length = 294100
++++
+
+## Um Teil von Skywire zu werden, kannst du deinen eigenen Skyminer bauen. Alles was du brauchst, sind einige Fähigkeiten und die folgenden Hardware:
+
+* x1 Mainboard (~ $ 100)
```
Comment (SkycnTrnsltns) [inline content/infographics/Build-A-DIY-Skycoin-Miner.de.md]: 1 x in German instead of x1 etc.

**Comment by kietob** [comment]
@gz-c : could you please rebase it?

**Comment by gz-c** [comment]
I am not able to. If you can't just ignore it. After it gets approval I will merge it and it should be fine

**Pull request merged**